quiver_engine/server.py

```diff
@@ -93,9 +93,21 @@ def get_layer_outputs(layer_name, input_path):
                 imsave(filename, deprocessed)
 
         return jsonify(output_files)
+
+    def decode_predictions(preds):
+        if len(preds.shape) != 2 or preds.shape[1] != len(classes):
+            raise ValueError('you need to provide same number of classes as model prediction output ' + \
```

Comment:
This is great, and correct, but since we already released Quiver with the imagenet preset, this change would break for people who are already using it to test things on imagenet. This is unfortunate, but what do you think about, at least for now:

When no `classes` is specified

- default to the imagenet decoding
- log an aggressive warning that says
      1.  no classes were provided therefore imagenet index is being used by default
      2. this behavior will be deprecated in the future